tests/components/unifi/test_sensor.py

```diff
@@ -155,8 +157,8 @@ async def test_remove_sensors(hass):
     controller.api.session_handler(SIGNAL_DATA)
     await hass.async_block_till_done()
 
-    assert len(hass.states.async_entity_ids("sensor")) == 2
-    assert len(hass.states.async_entity_ids("device_tracker")) == 1
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 2
+    assert len(hass.states.async_entity_ids(TRACKER_DOMAIN)) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)